LibreNMS/Interfaces/Polling/WirelessPolling.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace LibreNMS\Interfaces\Polling;
+use Illuminate\Support\Collection;
+
+interface WirelessPolling
```

Comment:
The interface should be called WirelessAccessPointPolling

Reply:
renamed